### PR TITLE
chore(RHTAPWATCH-638): add initial files

### PR DESCRIPTION
**Pull request by srivickynesh**
Create initial pipfile and github actions

### CODE REVIEW

**Review by yftacherzog — DISMISSED** [review]

File: .github/workflows/kube-linter.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+      - name: Setup Kustomize
+        uses: multani/action-setup-kustomize@v1
+        with:
+          version: 5.1.1
+
+      - name: Verify if the manifests were built
+        run: |
+            rm -rf auto-generated
+            ./build-manifests.sh
```
Comment (yftacherzog) [inline .github/workflows/kube-linter.yaml:28]: Where is this file coming from? Does it suppose to be inside our repo?

Comment (srivickynesh) [reply .github/workflows/kube-linter.yaml:28]: it s early to add kustomize build to run the kubelinter. will remove it for now

File: .github/workflows/kube-linter.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
+name: Check Kubernetes YAMLs with kube-linter
+
+on:
+  # Note that both `push` and `pull_request` triggers should be present for GitHub to consistently present kube-linter
+  # SARIF reports.
```
Comment (gbenhaim) [inline .github/workflows/kube-linter.yaml:5]: @srivickynesh can you drop this workflow and use this one instead https://github.com/redhat-appstudio/tenants-config/blob/main/.github/workflows/test.yaml ?

**Review by gbenhaim — CHANGES_REQUESTED** [review]

File: .github/workflows/kube-linter.yaml (hunk 2 above)
Comment (srivickynesh) [reply .github/workflows/kube-linter.yaml]: Done

**Comment by avi-biton** [comment]
lgtm

**Review by avi-biton — APPROVED** [review]

**Review by Omeramsc — APPROVED** [review]
LGTM

**Comment by eisraeli** [comment]
Please add a basic main file.

**Pull request merged**